package/dev_server.js

```
@@ -0,0 +1,20 @@
+// These are the raw shakapacker dev server config settings from the YML file with ENV overrides applied.
+const { isBoolean } = require("./utils/helpers")
+const config = require("./config")
```

ESLint

[error] Expected 1 empty line after require statement not followed by another require.

(import/newline-after-import)